modules/nvidia-gpu-admin-dashboard-using.adoc

```diff
@@ -0,0 +1,56 @@
+// Module included in the following assemblies:
+//
+// * monitoring/nvidia-gpu-admin-dashboard.adoc
+
+:_content-type: PROCEDURE
+[id="nvidia-gpu-admin-dashboard-using_{context}"]
+= Using the NVIDIA GPU administration dashboard
+
+After deploying the the OpenShift Console NVIDIA GPU plugin, log in to the OpenShift Container Platform web console using your login credentials to access the *Administrator* perspective.
+
+== Viewing the cluster GPU overview
+
+You can view the status of your cluster GPUs in the Overview page by selecting
+Overview in the Home section.
+
+The Overview page provides information about the cluster GPUs, including:
+
+* Details about the GPU providers
+* Status of the GPUs
+* Cluster utilization of the GPUs
+
+== Viewing the GPUs dashboard
+
+You can view the NVIDIA GPU administration dashboard by selecting GPUs
+in the Compute section of the OpenShift Console.
+
+Charts on the GPUs dashboard include:
+
+* GPU utilization
```

Comment:
This shows the ratio of time the graphics engine is active and is based on [this](https://docs.nvidia.com/datacenter/dcgm/latest/dcgm-api/dcgm-api-field-ids.html#c.DCGM_FI_PROF_GR_ENGINE_ACTIVE) metric.

Reply:
@mresvanis  I like the descriptions and associated metrics you provide. Can we do this for all of the charts?

Reply:
@mresvanis I added descriptions for the metrics you didn't provide descriptions for. It's my best guess, so please review carefully.